resources/assets/helpers.js

```diff
@@ -298,3 +256,77 @@ function stripExifData(image, dv = null) {
   // We can just create a blob with the original data.
   return new Blob([dataView], { type: 'image/jpeg' });
 }
+
+// DEPRECIATED: Uses getImageUrlFromPost() instead.
```

Comment:
This is the absolute nitpickiest comment, but this should be "deprecated". 🤓

Reply:
whoops! good catch haha